projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -608,16 +609,23 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
   }
 
   private onMouseLeave(): void {
-    if (this.tooltip) {
-      this.tooltip.hide();
-    }
-
-    this.eventListeners.forEach(listener => {
-      if (listener.event === ChartEvent.MouseLeave) {
-        listener.onEvent();
-      }
-    });
-
-    this.hideCrosshair();
+    this.tooltip?.hovered$
+      .pipe(
+        debounceTime(300),
```

Comment:
Why do we need the debounceTime?

Reply:
Added to provide a better user experience when hover over the graph with the cursor